docs/COMPILE.md

```diff
@@ -296,3 +298,13 @@ Signed-By: /usr/share/keyrings/box64-archive-keyring.gpg" | sudo tee /etc/apt/so
 sudo apt update
 sudo apt install box64-generic-arm -y
 ```
+
+## Cross-compiling 
+----
+### Manual Installation of Shared Libraries
+Copy the shared libraries folder (`x64lib` or `x86lib`) from the Box64 repository and transfer it to the target device. Ensure that the shared libraries are located in the appropriate search paths : 

```

Comment:
You should explain the installation of default build scripts and emphasize the case for cross-builds.